src/control/C4Control.h

```diff
@@ -54,18 +54,18 @@ class C4ControlPacket : public C4PacketBase
 	// allowed as direct/private control?
 	virtual bool Sync() const { return true; }
 
-	virtual void CompileFunc(StdCompiler *pComp);
+	void CompileFunc(StdCompiler *pComp) override;
 };
 
 #define DECLARE_C4CONTROL_VIRTUALS \
-  virtual void Execute() const; \
-  virtual void CompileFunc(StdCompiler *pComp);
+  void Execute() const override; \
+  void CompileFunc(StdCompiler *pComp) override;
```

Comment:
Oops, it wasn't clang-tidy. That's my handiwork. Should I revert it from this PR?